tools/src/Changelogs.ts

```diff
@@ -332,6 +343,125 @@ export class Changelog {
     throw new Error(`Cound't find '${type}' section.`);
   }
 
+  /**
+   * Insert an `VERSION_EMPTY_PARAGRAPH_TEXT` version section before first published version.
```

Comment:
In doc comments, I think we should rather use present tense in third person as it is to explain what the function **does** (third person).
```suggestion
   * Inserts an `VERSION_EMPTY_PARAGRAPH_TEXT` version section before first published version.
```